modules/archetypes/lib/role_definitions/role_definition_es_azure_platform_owner.tmpl.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "54875d31-4d56-572b-95c7-b74a682b30e0",
+    "type": "Microsoft.Authorization/roleDefinitions",
+    "apiVersion": "2018-01-01-preview",
+    "properties": {
+        "roleName": "Azure-Platform-Owner",
```

Comment:
Given this is equivalent to the built-in `Owner` I will discuss with the team as to whether we exclude this from the ALZ implementations. It would probably be better to update the CAF document to make this clearer that the "persona" for `Azure-Platform-Owner` maps to the existing `Owner` role. It seems silly to re-create a built-in role.